tests/queryset_pickle/tests.py

```diff
@@ -66,6 +66,20 @@ def test_model_pickle(self):
         self.assertEqual(original, reloaded)
         self.assertEqual(original.somefield, reloaded.somefield)
 
+    def test_model_pickle_with_ignoring_fields(self):
+        """
+        Test that developer can choose what fields he needs to pickle
+        by defining __getstate__ method.
+        """
+        queue_item = QueueItem.objects.create()
+        queue_item.put_in_queue()
```

Comment:
Do you need the `put_in_queue()` method? Would it be simpler to write `queue_item.order = 1` or something?